data-import/src/main/resources/folijet/data-import/features/vrn-matching.feature

```diff
@@ -0,0 +1,597 @@
+Feature: Test matching by VRN number and vendor reference number
+
+  Background:
+    * url baseUrl
+    * callonce login testAdmin
+    * def okapitokenAdmin = okapitoken
+
+    * callonce login testUser
+    * def okapitokenUser = okapitoken
+
+    * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': '*/*'  }
+    * def headersUserOctetStream = { 'Content-Type': 'application/octet-stream', 'x-okapi-token': '#(okapitokenUser)', 'Accept': '*/*'  }
+
+    * configure retry = { interval: 15000, count: 15 }
+
+  Scenario: FAT-2183 Match on VRN and update related Instance, Holdings, Item
```

Comment:
propose to move this "Scenario: FAT-2183 Match on VRN and update related Instance, Holdings, Item" test to the pol-vrn-matching.feature file to have tests for matching by POL and VRN in the same feature file